### ISSUE TITLE
Investigate why lib/styles/build doesn't meet lint spec

### ISSUE THREAD

**Issue by avkvirtru** [avkvirtru/virtuoso-design-system#39]
Hoping to avoid the need for `.stylelintignore` (to discourage folks from shoving their stylesheets in there rather than fixing lint issues).

See why CSS built by Style Dictionary doesn't match our current style lint's config. Could be an opportunity for that to grow. Or to fix Style Dictionary.

_Originally posted by @jrschumacher in https://github.com/avkvirtru/virtuoso-design-system/pull/37_

**Comment by jrschumacher** [comment]
From my perspective this is expected since they are being built. In a future state I'd want webpack to run the build process so we don't have to commit any build files.

**Comment by avkvirtru** [comment]
Good point. Just curious what the gaps are. In case the built CSS ever gets used manually in static sites or whatnot, I'd love to know how it could be better.
